watcher/watcher.go

```diff
@@ -240,9 +256,113 @@ func (w *Watcher) reportRoomsStatusesRoutine() {
 func (w *Watcher) watchRooms() error {
 	w.RemoveDeadRooms()
 	w.AutoScale()
+	w.AddUtilizationMetricsToRedis()
```

Comment:
Nice one! 
My idea was actually to add the metrics inside `w.AutoScale()` but this way is much cleaner.